index.html

```diff
@@ -9,63 +9,77 @@
     <link rel="stylesheet" href="styles/styles.css" />
   </head>
   <body>
-    <div class="section">
-      <h2 class="section_title">What do we offer</h2>
-      <hr class="line" />
-      <div class="img_card_container">
-        <div class="img_card">
-          <img class="img_icon" src="images/headfones.jpg" alt="headfones" />
-          <span class="text">Support 24/7</span>
-        </div>
-        <div class="img_card">
-          <img class="img_icon" src="images/chat.jpg" alt="chat" />
-          <span class="text">Communicate directly</span>
-        </div>
-        <div class="img_card">
-          <img
-            class="img_icon"
-            src="images/book_online.jpg"
-            alt="book_online"
-          />
-          <span class="text">Book online</span>
-        </div>
-        <div class="img_card">
-          <img class="img_icon" src="images/like.jpg" alt="like" />
-          <span class="text">Real guest reviews</span>
-        </div>
+  <div class="svg_sprite">
+    <svg display="none">
+      <symbol id="logo" viewBox="0 0 205 40">
+      <svg width="205" height="40" viewBox="0 0 205 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
![image](https://user-images.githubusercontent.com/79144475/227652684-49a4401f-e859-4087-abdf-ed6dbba7f989.png)

Reply:
You can immediately write in the sprite only once for all icons 
xmlns="http://www.w3.org/2000/svg"